docs/manual/pubsub.md

```diff
@@ -38,6 +38,8 @@ stream of messages where the first element indicates the type of
 message. The commands that are allowed in the context of a subscribed
 client are `SUBSCRIBE`, `SSUBSCRIBE`, `SUNSUBSCRIBE`, `PSUBSCRIBE`, `UNSUBSCRIBE`, `PUNSUBSCRIBE`, `PING`, `RESET`, and `QUIT`.
 
+In particular, the operation of the `UNSUBSCRIBE` and `PUNSUBSCRIBE` commands in redis-cli has limitations, so keep in mind.
+
 Please note that `redis-cli` will not accept any commands once in
 subscribed mode and can only quit the mode with `Ctrl-C`.
```

Comment:
@zuiderkwast  Thanks again for your review.

I combined the sentences as below and created two candidates. 
Please confirm. :)

(1)
If the client using redis-cli is in subscribed mode, redis-cli will not accept the commands once and can only quit the mode with Ctrl-C. In particular, the commands such as `UNSUBSCRIBE` and `PUNSUBSCRIBE` cannot be used.

or

(2)
When a client using redis-cli is in subscription mode, commands such as `UNSUBSCRIBE` and `PUNSUBSCRIBE` cannot be used because redis-cli will not accept the commands once more and can only quit the mode with Ctrl-C.

Reply:
Both are good, but maybe start with "Please note" as before.

Reply:
@zuiderkwast  Thanks for checking.

Please note that when a client using redis-cli is in subscription mode, commands such as 'UNSUBSCRIBE' and 'PUNSUBSCRIBE' cannot be used because redis-cli will not accept the commands once more and can only quit the mode with Ctrl-C.